core/ir/x86/decode_table.c

```diff
@@ -1106,7 +1106,7 @@ const instr_info_t * const op_instr[] =
     /* OP_wrgsbase      */   &mod_extensions[17][1],
```

Comment:
Please add a PR description (convention is to always have one); maybe elaborate on what RDPID does -- it may well prove useful to instrumentation if it's faster than RDTSCP.